pkg/webhook/seed/validation.go

```diff
@@ -190,6 +192,16 @@ func (v *validator) validate(ctx context.Context, obj runtime.Object, isDelete b
 		if _, exists := subject.Spec.EtcdBackupRestore.Destinations[subject.Spec.EtcdBackupRestore.DefaultDestination]; !exists {
 			return fmt.Errorf("invalid etcd backup configuration: default destination %q does not exist", subject.Spec.EtcdBackupRestore.DefaultDestination)
 		}
+
+		for _, dest := range subject.Spec.EtcdBackupRestore.Destinations {
+			if dest.Credentials != nil {
+				etcdBackupSecret := corev1.Secret{}
+				if err := seedClient.Get(ctx, types.NamespacedName{Name: dest.Credentials.Name,
+					Namespace: dest.Credentials.Namespace}, &etcdBackupSecret); err != nil {
+					return fmt.Errorf("invalid etcd backup configuration: invalid etcd backup configs for destination %v: %w", dest.Credentials.Name, err)
```

Comment:
> invalid etcd backup configuration: invalid etcd backup configs for destination %v

sounds a bit stuttery. WDYT about

> invalid etcd backup configuration: invalid destination %s: %w

and replace %s with the name of the destination, not the same of the credential Secret.

Reply:
There is no name for a destination

Reply:
That's why I used the credential ;)